pkgs/top-level/all-packages.nix

```
@@ -29411,8 +29411,13 @@ in
 
   yaxg = callPackage ../tools/graphics/yaxg {};
 
-  yuzu = libsForQt5.callPackage ../misc/emulators/yuzu {
-    stdenv = gcc10Stdenv;
+  yuzu-mainline = import ../misc/emulators/yuzu {
+    branch = "mainline";
+    inherit pkgs;
```

```suggestion
    inherit (pkgs) libsForQt5 fetchFromGitHub;
```

I don't think we need to inherit all pkgs here.